### DISCUSSION TITLE
In the FR documentation, lot of links ends with "%5C%22" (i.e. \" )

### DISCUSSION THREAD

**Discussion by denis-migdal** [mdn/mdn-community#317] [Platform] [answered]
Hi,

I noticed that lot of URLs in the documentation are ending with %5C%22 (i.e. \" ).

I assume there is a server-side issue with the generation of the URL for some pages ?

Cordially,

**Comment by schalkneethling** [comment]
Pinging @mdn/core-dev - Thank you for reporting, @denis-migdal

**Comment by denis-migdal** [comment]
I just noticed some URL ending with "%255C%2522" which is "%5C%22" escaped.

I am also not sure if your ping to the core-dev worked as it seems different to the ping you made to me.

**Reply by Josh-Cena** [reply]
@denis-migdal If I understand you correctly—pings that notify you appear in a yellow background. Pings that are irrelevant to you aren't.

**Reply by denis-migdal** [reply]
Pings that notify me indeed appear in a yellow background, but they are also a link with a "pop-up"/"tooltip" that appear on hover.
The ping to the core-dev doesn't seem to be a link and nothing shows when hovered. So I wondered if the ping worked.

I am not quite used to github discussions but it seems strange to me.

**Reply by Josh-Cena** [reply]
<img width="512" alt="image" src="https://user-images.githubusercontent.com/55398995/212520934-bb561c67-4afd-4573-9adc-fad67bd46adf.png">

Here's what I see. Because `@mdn/core-dev` is a team, there's a different card for it.

**Reply by yin1999** [reply]
Exactly, the team's ping is only visible to members of the organization. So you could not see the card, but you could see [this](https://developer.mozilla.org/en-US/docs/MDN/Community/Roles_teams#teams) to know the responsibility of each team.

**Comment by SphinxKnight** [comment]
@denis-migdal could you please provide some examples (whether it's screenshots or URLs where the issue occurs)?

**Comment by bsmth** [comment, marked as answer]
I can't find any hints that this is the case through code search, so I'm going to close for now, it's going to be extremely hard to dig deeper into this, and we should assume that this is fixed unless we see another report. Thanks all :)
